Appdata and runtime related changes

yakushabb opened the pull request:
- runtime: Update runtime to 46
- appdata: Add vcs-browser URL
- appdata: Add missing developer name

yakushabb committed:
Appdata and runtime related changes

- runtime: Update runtime to 46
- appdata: Add vcs-browser URL
- appdata: Add missing developer name

whot commented:
```
5/6 appstream-util validate-relax           FAIL     0.52 s (exit status 1)

--- command ---
23:31:59 /usr/bin/appstream-util validate-relax org.freedesktop.Tuhi.appdata.xml
--- stdout ---
org.freedesktop.Tuhi.appdata.xml: FAILED:
• tag-invalid           : <url> type invalid [unknown]
--- stderr ---
Validation of files failed
-------
```

Is this a side-effect of the CI using Ubuntu 20.04? If so, can you rebase on top of master, let's see if #304 fixed this.

yakushabb committed:
meson: Use appstreamcli

Use appstreamcli to validate appdata

More information: https://docs.flathub.org/docs/for-app-authors/metainfo-guidelines/#validation

ghost commented:
I've added another commit 

> Use appstreamcli to validate appdata
> 
> More information: https://docs.flathub.org/docs/for-app-authors/metainfo-guidelines/#validation

whot commented:
Thanks!
